taprpc/universerpc/universe.proto

```diff
@@ -145,6 +145,12 @@ message MerkleSumNode {
     // exposed via iterative queries
 }
 
+enum ProofType {
+    UNSPECIFIED = 0;
```

Comment:
Due to a weird constraint in gRPC the individual enum value names must be unique across a whole RPC package. So we couldn't use `TRANSFER` within another enum (even if the enum itself is called differently) without getting a conflict. That's why we normally prefix each enum value name by its enum name. For example: `PROOF_TYPE_TRANSFER`.

Reply:
Good call :+1: I've addressed this here https://github.com/lightninglabs/taproot-assets/pull/540